docs/3.0/manage/cloud/connect-to-cloud.mdx

```diff
@@ -82,9 +82,18 @@ prefect config set PREFECT_API_URL="https://api.prefect.cloud/api/accounts/[ACCO
 prefect config set PREFECT_API_KEY="[API-KEY]"
 ```
 
-When you're in a Prefect Cloud workspace, you can copy the `PREFECT_API_URL` value directly from the page URL.
+<Note>
+**Find account ID and workspace ID in the browser**
+
+When authenticated to your Prefect Cloud workspace in the browser, you can get the account ID and workspace ID for the `PREFECT_API_URL` string from the page URL.
+
+For example, if you're on the dashboard page, the URL looks like this: 
+
+https://app.prefect.cloud/account/[ACCOUNT-ID]/workspace/[WORKSPACE-ID]/dashboard
+</Note>
+
+The example above configures `PREFECT_API_URL` and `PREFECT_API_KEY` in the default profile. 
```

Comment:
manually extracting these 2 IDs out of the url in the browser feels not ideal, that might be the best we have now, but itd be cool if we could just copy this value somewhere in My Profile or something